tests/test_all_rules.py

```diff
@@ -870,7 +884,7 @@ def test_integration_guide(self):
             note_str = integration_notes.get(integration)
 
             if note_str:
-                self.assert_(rule.contents.data.note, f'{self.rule_str(rule)} note required for config information')
+                self.assertIsNotNone(rule.contents.data.note, f'{self.rule_str(rule)} note required for config information')  # noqa: E501
```

Comment:
```suggestion
                error_message = f'{self.rule_str(rule)} note required for config information'
                self.assertIsNotNone(rule.contents.data.note, error_message)
```

Nit we could not ignore linting here by splitting into two lines. If we switch to ruff enforcement, this will be best practice (defining the error message on a different line).